chia/util/initial-config.yaml

```diff
@@ -550,3 +546,27 @@ wallet:
 
   # Interval to resend unconfirmed transactions, even if previously accepted into Mempool
   tx_resend_timeout_secs: 1800
+
+simulator:
+  # The full node simulator (if run) will run on this port
+  port: 8444
+
```

Comment:
If the config variables are already in the full node section, there is no need to repeat them here

Reply:
These will be changed by the wizard later, and they allow the existing simulator to continue working

Reply:
The existing simulator which is designed to work with the gui expects the simulator to be at the default ports, so we will have the wizard change the ports to the testnet ports once it is run.

Reply:
can we have the GUI read the config instead? I think the way it works with existing GUI simulator is kind of bad because you can only run one at a time

Reply:
I removed the ports from it, should be good now